Content.Server/GameObjects/Components/Research/MaterialStorageComponent.cs

```diff
@@ -0,0 +1,88 @@
+using System.Collections.Generic;
+using Content.Shared.GameObjects.Components.Research;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.Network;
+using Robust.Shared.Serialization;
+
+namespace Content.Server.GameObjects.Components.Research
+{
+    public class MaterialStorageComponent : SharedMaterialStorageComponent
+    {
+        private Dictionary<string, int> _storage = new Dictionary<string, int>();
+        protected override Dictionary<string, int> Storage => _storage;
```

Comment:
You can just re-define the auto property like so:

```cs
protected override Dictionary<string, int> Storage { get; private set; } = new Dictionary<string, int>();
```

Reply:
Hm, it didn't let me set it to private set, but it's okay anyway.
Fixed it.